adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/view.jsp

```diff
@@ -258,6 +256,57 @@ PortletURL portletURL = renderResponse.createRenderURL();
 	}
 </aui:script>
 
+<aui:script require="metal-dom/src/dom">
+	var dom = metalDomSrcDom.default;
+
+	Liferay.on('start_optimizing', (event) => {
```

Comment:
This is dangerous, and we are not cleaning up, so that's bound to break if you navigate...